version.json

```diff
@@ -0,0 +1,6 @@
+{
+  "major": 1,
+  "minor": 0,
+  "patch": 1,
+  "rc": 1
+}
```

Comment:
Do we really need this file? The version in the package.json should be sufficient.